solr-updater/src/main/java/dk/dbc/search/solrdocstore/updater/BusinessLogic.java

```diff
@@ -255,19 +280,81 @@ public void attachedResources(JsonNode sourceDoc) {
         }
     }
 
+    /**
+     * Set field is any is set to true
+     *
+     * @param indexKeys                The bibliographic part of the json from
+     *                                 solr-doc-store
+     * @param fieldName                Where to store the computed value
+     * @param values                   json values (object of agency ->
+     *                                 true/false) (solr-doc-store
+     *                                 attachedResources -> resource)
+     * @param setFalseOverrideExisting if false override existing value ?
+     */
+    private void anyTrueSet(JsonNode indexKeys, String fieldName, JsonNode values, boolean setFalseOverrideExisting) {
+        Optional<String> isTrue = fieldsStream(values)
+                .filter(e -> e.getValue().asBoolean(false) == true) // any that has the values true
+                .map(Entry::getKey)
+                .findAny();
+        if (isTrue.isPresent()) {
+            log.trace("key {} (anyTrue), {} is true", fieldName, isTrue.get());
+            setField(indexKeys, fieldName, "true");
+        } else if (setFalseOverrideExisting) {
+            setField(indexKeys, fieldName, "false");
+        }
+    }
+
+    /**
+     * Set field is all are set to true
```

Comment:
=> Set field if

Måske: Set a given field to the value true, if all the values passed are true.